educator/src/Dscp/Educator/DB/Instances.hs

```diff
@@ -48,6 +49,14 @@ instance FromField (TYPE) where \
     fromField field ty = \
         leftToPanic . fromByteArray @(TYPE) @ByteString <$> fromField field ty
 
+#define JsonInstanceEnc(TYPE) \
+instance IsPgValue (TYPE) where \
+    sqlValueSyntax = sqlValueSyntax . Aeson.encode
+
+#define JsonInstanceDec(TYPE) \
+instance FromField (TYPE) where \
+    fromField field ty = leftToPanic . Aeson.eitherDecode <$> fromField field ty
```

Comment:
Not sure that creating a macro for a single use case is a good thing to do, but whatever.